src/router/summary_router.py

```
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, HTTPException, Depends
+from Service.meeting_summary import MeetingSummarizer
+from router.Schema.response import SummaryResponse
+
+router = APIRouter()
+
+def get_summarizer():
+    return MeetingSummarizer()
+
+@router.post("/summarize", response_model=SummaryResponse)
+async def summarize_meeting(summarizer: MeetingSummarizer = Depends(get_summarizer)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)